src/linter/root.go

```diff
@@ -1038,75 +1003,57 @@ func (d *RootWalker) parsePHPDoc(doc string, actualParams []node.Node) (phpDocPa
 
 	var curParam int
 
-	lines := strings.Split(doc, "\n")
-	for idx, ln := range lines {
-		ln = strings.TrimSpace(ln)
-		if len(ln) == 0 {
-			phpDocErrors = append(phpDocErrors, fmt.Sprintf("empty line %d", idx))
-			continue
-		}
-
-		ln = strings.TrimPrefix(ln, "/**")
-		ln = strings.TrimPrefix(ln, "*")
-		ln = strings.TrimSuffix(ln, "*/")
-
-		if strings.Contains(ln, "@deprecated") {
-			parts := strings.Split(ln, "@deprecated")
+	for _, part := range phpdoc.Parse(doc) {
+		if part.Name == "deprecated" {
 			result.info.Deprecated = true
-			result.info.DeprecationNote = strings.TrimSpace(parts[1])
+			// strings.Join here is unfortunate.
```

Comment:
I guess we could retain the original string somewhere as well to avoid having to join back the string